misk-admin/src/main/kotlin/misk/web/dashboard/BaseDashboardModule.kt

```diff
@@ -14,8 +18,18 @@ class BaseDashboardModule(
   private val isDevelopment: Boolean
 ): KAbstractModule() {
   override fun configure() {
-    // Install base dashboard support
-    install(DashboardModule())
+    // Setup multibindings for dashboard related components
+    newMultibinder<DashboardTab>()
```

Comment:
I inlined the old `DashboardModule` since it was only installed here and I wanted to use that name for other purposes going forward.